### DISCUSSION TITLE
Clear the cache of running projects

### DISCUSSION THREAD

**Discussion by hgjo-nanum** [vercel/next.js#70442] [Help]
### Summary

Hello! I have a question.
Is there a way to clear cache in a running next.js project?
I don't want to stop the running server, delete the .next folder, and rebuild.
I'm wondering how to clear it without stopping the server.

**Comment by kimpuro** [comment]
Try using a shell script that periodically deletes the .next/cache directory.
```
#!/bin/bash

# Set the path for .next/cache
CACHE_DIR="/path/to/your/project/.next/cache"

# Check if the cache folder exists
if [ -d "$CACHE_DIR" ]; then
  # Delete the cache folder
  rm -rf "$CACHE_DIR"
  echo "$(date): .next/cache folder has been deleted." >> /path/to/your/logfile.log
else
  echo "$(date): .next/cache folder does not exist." >> /path/to/your/logfile.log
fi
```
Afterwards, give the script execute permission and make it run periodically.
`chmod +x /path/to/clear_next_cache.sh`
`crontab -e`
`0 0 * * * /path/to/clear_next_cache.sh`

This script simply deletes the .next/cache directory. 
I'm not sure what would happen if you force-delete the cache of a running next application.

**Reply by hgjo-nanum** [reply]
Thanks for your reply.
So, if I delete all the cache files while the server is running, there won't be any problems?

**Reply by kimpuro** [reply]
When I deleted the entire cache folder, there was no major problem, although I got a webpack cache related error. 
However, I think deleting the fetch-cache folder will be fine.
